sdks/go/pkg/beam/core/util/ptransform/ptransform.go

```diff
@@ -0,0 +1,19 @@
+package ptransform
+
+import "context"
+
+var key = struct{ Key string }{Key: "beam:ptransform"}
```

Comment:
This is typically done using a type definition of a string rather than an anonymous struct. This just seems unusual without any motivation for using a more complicated type.

Reply:
Done, it had been a while since I read the context blogpost (https://blog.golang.org/context) and mis-remembered.